Taskfile.yml

```diff
@@ -319,10 +319,25 @@ tasks:
   mongosh:
     desc: "Run MongoDB shell (`mongosh`)"
     cmds:
-      - >
-        docker compose exec mongodb mongosh
-        'mongodb://host.docker.internal:27018/test?tls=true&tlsCAFile=/etc/certs/rootCA.pem&heartbeatFrequencyMS=300000'
-        --verbose --eval 'disableTelemetry()' --shell
+      - task: mongosh-plain
+      - task: mongosh-scram
+
+  mongosh-plain:
+    cmds:
+      - cmd: >
+          docker compose exec mongodb mongosh
+          'mongodb://username:password@host.docker.internal:27018/?authMechanism=PLAIN&tls=true&tlsCAFile=/etc/certs/rootCA.pem&heartbeatFrequencyMS=300000'
+          --verbose --eval 'disableTelemetry()' --shell /mongosh/test.js
```

Comment:
We need to add `/mongosh/test.js`. 

```
$ task mongosh-plain
task: [mongosh-plain] docker compose exec mongodb mongosh 'mongodb://username:password@host.docker.internal:27018/?authMechanism=PLAIN&tls=true&tlsCAFile=/etc/certs/rootCA.pem&heartbeatFrequencyMS=300000' --verbose --eval 'disableTelemetry()' --shell /mongosh/test.js

Current Mongosh Log ID: 63a51394e3fdd7a0870bc7c8
Connecting to:          mongodb://<credentials>@host.docker.internal:27018/?authMechanism=PLAIN&tls=true&tlsCAFile=%2Fetc%2Fcerts%2FrootCA.pem&heartbeatFrequencyMS=300000&directConnection=true&appName=mongosh+1.6.1
Using MongoDB:          6.0.42
Using Mongosh:          1.6.1

For mongosh info see: https://docs.mongodb.com/mongodb-shell/

------
   The server generated these startup warnings when booting
   2022-12-23T02:33:56.77Z: Powered by FerretDB v0.7.1-77-ge6e9ac2 and PostgreSQL 15.1.
   2022-12-23T02:33:56.77Z: Please star us on GitHub: https://github.com/FerretDB/FerretDB.
   2022-12-23T02:33:56.77Z: The telemetry state is undecided; the first report will be sent soon. Read more about FerretDB telemetry and how to opt out at https://beacon.ferretdb.io.
------

------
   Enable MongoDB's free cloud-based monitoring service, which will then receive and display
   metrics about your deployment (disk utilization, CPU, operation statistics, etc).
   
   The monitoring data will be available on a MongoDB website with a unique URL accessible to you
   and anyone you share the URL with. MongoDB may use this information to make product
   improvements and to suggest MongoDB products and deployment options to you.
   
   To enable free monitoring, run the following command: db.enableFreeMonitoring()
   To permanently disable this reminder, run the following command: db.disableFreeMonitoring()
------

Telemetry is now disabled.
Loading file: /mongosh/test.js
Error: ENOENT: no such file or directory, open '/mongosh/test.js'

```

Reply:
Maybe just use the empty file`build/legacy-mongo-shell/test.js`?
```
chi@alma:~/go/src/github.com/chilagrow/FerretDB$ git diff
diff --git a/docker-compose.yml b/docker-compose.yml
index 76add6d..5e8ef59 100644
--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -60,6 +60,7 @@ services:
     volumes:
       - ./build/certs:/etc/certs
       - ./build/mongod.conf:/etc/mongod.conf
+      - ./build/legacy-mongo-shell/test.js:/mongosh/test.js
 
   # for test scripts
   legacy-mongo-shell:
```

Reply:
`/mongosh/test.js` should not be used – tests scripts should use legacy shell, not mongosh. I guess that change is a result of mis-merge

Reply:
Fixed.